apollo-router/tests/set_context.rs

```diff
@@ -313,6 +313,8 @@ fn setup_from_mocks(
     configuration: serde_json::Value,
     mocks: &[(&'static str, &'static str)],
 ) -> TestHarness<'static> {
+    let _ = rustls::crypto::aws_lc_rs::default_provider().install_default();
+
```

Comment:
@goto-bus-stop This looks like an unintentional change introduced by the merge.
Is it correct?

Reply:
Tests failed without it. We have to do this in several places after the hyper 1.0 upgrade on dev, but it's a bit weird that these tests were passing on dev before :/

Reply:
It's because tonic changed. We need to call this.